pkg/report/source_filter.go

```diff
@@ -128,9 +128,11 @@ func NewSourceFilter(pods PodClient, jobs JobClient, validations []SourceValidat
 	return &SourceFilter{pods: pods, jobs: jobs, validations: validations}
 }
 
-var controller = []string{"ReplicaSet", "DaemonSet", "CronJob", "Job", "Job", "StatefulSet"}
+var podControllers = []string{"ReplicaSet", "DaemonSet", "Job", "StatefulSet"}
 
-func Uncontrolled(owner []metav1.OwnerReference) bool {
+var jobControllers = []string{"CronJob"}
+
+func Uncontrolled(owner []metav1.OwnerReference, controllers []string) bool {
```

Comment:
this is potentially backwards incompatible, please tell me if backwards compatibility for this function should be preserved